src/strings/en-us.json

```diff
@@ -1716,5 +1716,8 @@
     "OptionExtractTrickplayImage": "Enable trickplay image extraction",
     "ExtractTrickplayImagesHelp": "Trickplay images are similar to chapter images, except they span the entire length of the content and are used to show a preview when scrubbing through videos.",
     "LabelExtractTrickplayDuringLibraryScan": "Extract trickplay images during the library scan",
-    "LabelExtractTrickplayDuringLibraryScanHelp": "Generate trickplay images when videos are imported during the library scan. Otherwise, they will be extracted during the trickplay images scheduled task. If generation is set to non-blocking this will not affect the time a library scan takes to complete."
+    "LabelExtractTrickplayDuringLibraryScanHelp": "Generate trickplay images when videos are imported during the library scan. Otherwise, they will be extracted during the trickplay images scheduled task. If generation is set to non-blocking this will not affect the time a library scan takes to complete.",
+    "LabelSaveTrickplayLocally": "Save trickplay images next to media",
+    "LabelSaveTrickplayLocallyHelp": "Saving trickplay images into media folders will put them next to your media for easy migration and access.",
+    "ReplaceTrickplayImages": "Replace existing trickplay images"
```

Comment:
Let's add new strings in alphabetical order.
Here
https://github.com/jellyfin/jellyfin-web/blob/18297b499a78036c52effd977d057a41c132f7e4/src/strings/en-us.json#L823-L824
and here
https://github.com/jellyfin/jellyfin-web/blob/18297b499a78036c52effd977d057a41c132f7e4/src/strings/en-us.json#L1361-L1362